src/NLog/Targets/EventLogTarget.cs

```diff
@@ -194,10 +194,8 @@ public int MaxMessageLength
         /// <param name="installationContext">The installation context.</param>
         public void Install(InstallationContext installationContext)
         {
-            var fixedSource = GetFixedSource();
-
-            //always throw error to keep backwardscomp behavior.
-            CreateEventSourceIfNeeded(fixedSource, true);
+            // always throw error to keep backwards compatible behavior.
+            CreateEventSourceIfNeeded(GetFixedSource(), true);
```

Comment:
Why is this changed?

Reply:
1) Improves readability: "backwardscomp" → "backwards compatible".
2) Inlined temporary variable to simplify the method.